platforms/codesandbox/deploy.ts

```diff
@@ -36,16 +36,19 @@ async function fetchWithPuppeteer(endpoint) {
   const page = await browser.newPage()
   await page.setDefaultNavigationTimeout(0)
   await page.goto(endpoint)
-  await page.waitForTimeout(10000)
+  await page.waitForTimeout(20_000)
   const screenshot = await page.screenshot()
   fs.writeFileSync('image.png', screenshot as Buffer)
   await browser.close()
 
   const r = await fetch(endpoint)
+  const body = await r.text()
   try {
-    await r.json()
+    const bodyAsJSON = JSON.parse(body)
```

Comment:
When you did `r.text()` you can not use `r.json()` any more to get it already parsed?

Reply:
Here it's a custom implementation, so it's a bit different but normally yes only once

Reply:
Or you will get an error